1-bootstrap/README.md

```diff
@@ -61,6 +61,27 @@ You can now deploy the common environment for these pipelines.
 
 If you receive any errors or made any changes to the Terraform config or `terraform.tfvars`, re-run `terraform plan` before you run `terraform apply`.
 
+### Updating `backend.tf` files on the repository
+
```

Comment:
Just a thought, might want to include "why" someone would want to do this procedure.

Reply:
added this description:

_"Within the repository, you'll find `backend.tf` files that define the GCS bucket for storing the Terraform state. By running the commands below, instances of `UPDATE_ME` placeholders in these files will be automatically replaced with the actual name of your GCS bucket."_